receiver/statsdreceiver/protocol/statsd_parser.go

```diff
@@ -61,10 +63,17 @@ type statsDMetricdescription struct {
 	labels           attribute.Distinct
 }
 
-func (p *StatsDParser) Initialize(enableMetricType bool) error {
+func (p *StatsDParser) Initialize(enableMetricType bool, sendTimerHistogram string) error {
 	p.gauges = make(map[statsDMetricdescription]pdata.InstrumentationLibraryMetrics)
 	p.counters = make(map[statsDMetricdescription]pdata.InstrumentationLibraryMetrics)
+	p.timers = make([]pdata.InstrumentationLibraryMetrics, 0)
 	p.enableMetricType = enableMetricType
+	switch sendTimerHistogram {
+	case "gauge":
+		p.sendTimerHistogram = sendTimerHistogram
+	default:
+		p.sendTimerHistogram = "gauge"
```

Comment:
It seems that no matter what value `send_timer_histogram` will be set, `p.sendTimerHistogram = "gauge"`. Why do we need this switch case?

Reply:
Also, what if the customer set `send_timer_histogram = "counter"` or some other string?

Reply:
I will have another PR later for using OTLP `summary`, I just left the switch here for later usage.

Reply:
Also, we may need to support other types such as `histogram`

Reply:
The default value in this switch function still make no sense to me, if the customer set `send_timer_histogram = "histogra" ` (which should be `histogram`), do we still set `send_timer_histogram = "counter"` ?

Reply:
Yes, because for now, we only support `gauge`. Let me revise the `README` saying about that.

Reply:
I know that you put the switch function here for another type `summary`. And that's the reason I see a potential issue.
For the design of your code, if customer have a typo for the `send_timer_histogram` in the config file (like they have a typo like:  "histogarm" or "histogra"), their original purpose is to use `histogram` , you will make they to choose to use `gauge` without noticing there are some errors in their config file.  Customers will be confused because what they want to see is the result of using `histogram`, it will be difficult for them to find the root issue.  This is a bad customer experience.
My suggestion is:
We can have some error check for the `send_timer_histogram` in the config file. If customer have any typo, we should let them know instead of using `gauge` by default.

Reply:
Sure, make sense. I will add some error handlings.